platform/src/presentations/select_response/mod.rs

```diff
@@ -0,0 +1,115 @@
+#![allow(non_snake_case)]
+use crate::{components::button::Button, prelude::*};
+use dioxus::prelude::*;
+
+#[derive(PartialEq, Props, Clone)]
+pub struct SelectResponseProps {
+    lang: Language,
+    title: String,
+}
+
+pub mod controller;
+pub mod i18n;
+
+#[component]
+pub fn SelectResponsePage(props: SelectResponseProps) -> Element {
+    let mut ctrl = controller::Controller::init(props.title.clone());
+    let survey = ctrl.get_survey();
+    let question_list = survey.questions.len() as u64;
+    let translates = i18n::translate(props.lang.clone());
+    let question_list_info_first = translates.question_list_info_first.clone();
+    let question_list_info_second = translates.question_list_info_second.clone();
+    let question_list_info = question_list_info_first
+        + question_list.to_string().as_str()
+        + question_list_info_second.as_str();
+
+    rsx! {
+        div {
+            class: "flex flex-col w-full h-full justify-start items-center",
+            div {
+                class: "flex flex-col max-w-[1200px] min-w-[600px] w-full justify-start items-start mt-[55px] px-[50px]",
+                div {
+                    class: "flex flex-row w-full justify-end items-end mb-[20px]",
+                    Button {
+                        button_text: translates.temporary_save,
+                        onclick: move |_| {},
+                        class: "flex flex-row w-[200px] h-[50px] bg-[#1e5eaf]",
+                    }
+                }
+                div {
+                    class: "flex flex-row w-full h-[110px] rounded-[10px] bg-white mb-[10px]",
+                    div {
+                        class: "flex flex-row w-full h-[110px] items-center justify-start text-[#2168c3] font-semibold text-[30px] pl-[30px]",
+                        {survey.survey.title}
```

Comment:
```suggestion
                        "{survey.survey.title}"
```
이렇게 하면 move 없이 사용할 수 있어요.

Reply:
따옴표 넣는 방식으로 수정했습니다!